### ISSUE TITLE
[HELP]: How to play multiple times same audio recording

### ISSUE THREAD

**Issue by mihailovukojicic** [Canardoux/flutter_sound#1138]
## Package version: 9.2.13
------------

I need Help for :

- Other ?

-------------------------------------
## Here is my question :

I am using your package for multiplatform web and mobile, and currently, I am working on my Flutter web section, so I was wondering, once the user makes a recording, I can only play that sound once, is it possible to play it multiple times? One more question regarding audio playback: if I create a custom paint design for displaying audio recordings in the form of a wave, can I achieve user interaction with an audio wave and skip to certain parts of the recording? Thanks in advance.

**Comment by Larpoux** [comment]
> once the user makes a recording, I can only play that sound once, is it possible to play it multiple times?

There is no reason not to be able to play your sound multiple times, during an audio session.
But on a web app, you don’t have access to a real file system. With Flutter Sound, files are emulated with Blob objects. If you need to be able to play a file later, during a new audio session, you must save your Blob object so that you can restore it later.

> , can I achieve user interaction with an audio wave and skip to certain parts of the recording? 

Flutter Sound is a plugin used to record and play audio. It is not used to analyze, handle and manipulate static audio buffers. Other plugins like Flutter FFmpeg can be used for that.

**Comment by mihailovukojicic** [comment]
Yes, that's what I was trying to do, after recording in the same session, I can only play it once, and then the second time, it's not working.
Thanks for the quick response.

**Comment by mihailovukojicic** [comment]
Sorry, I have another question: how do I extract recorded file length, I am using `Codec.pcm16WAV`.
